march_hardware/CMakeLists.txt

```diff
@@ -54,6 +54,8 @@ add_library(${PROJECT_NAME}
     include/${PROJECT_NAME}/ethercat/sdo_interface.h
     include/${PROJECT_NAME}/ethercat/slave.h
     include/${PROJECT_NAME}/imotioncube/actuation_mode.h
+    include/${PROJECT_NAME}/imotioncube/motor_controller.h
+    include/${PROJECT_NAME}/imotioncube/motor_controller_state.h
```

Comment:
Why add the "general" motor controller to the `imotioncube` module? Perhaps create a motor controller module which contains the imotioncube module?